gsoc/2013.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <title>Systers Open Source Projects by Systers</title>
-
+    
```

Comment:
@Saichethan these changes are unnecessary, mostly introduced by IDE's reformatting. Try not to push these commits by using `git add -p` to do a selective addition.

Reply:
Actually, I Made changes to meta description in this branch but to make them independent I removed them later.

Reply:
@sara-02 , I think similar issue was opened, #80

Reply:
ya @sara-02 ,  RGSoC and Outreachy  names have changed

Reply:
Should I change something else?

Reply:
@Saichethan No we are good with this as of now.

Reply:
@sara-02 , I think I changed let me know if there is anything I should do.